packages/fx-core/templates/plugins/resource/aad/auth/bot/README.md

```diff
@@ -4,7 +4,19 @@ Microsoft Teams provides a mechanism by which an application can obtain the sign
 
 For a bot application, user can invoke the AAD consent flow to obtain sso token to call Graph and other APIs. 
 
-# Changes to your project
+<h2>Contents </h2>
+
+* [Changes to your project](#1)
+* [Update code to Use SSO for Bot](#2)
+  * [Set up the AAD redirects](#2.1)
+  * [Update your business logic](#2.2)
+  * [(Optional) Add a new sso command to the bot](#2.3)
+* [Update code to Use SSO for Message Extension](#3)
+* [Debug your application](#4)
+* [Customize AAD applications](#5)
+* [Trouble Shooting](#6)
+
```

Comment:
Minor suggestion, do you want to use this format for hyper link in README? Doesn't matter that much.
![image](https://user-images.githubusercontent.com/11220663/192729327-0bc1492b-95dc-423f-a71d-9b1048066536.png)

Reply:
https://github.com/OfficeDev/TeamsFx/tree/wenyt/me/ttk/packages/fx-core/templates/plugins/resource/aad/auth/bot
This url could work fine.
I perfer to use this, maybe there title update but forget to update the content, the hyper link get invalid